eng/scripts/Language-Settings.ps1

```diff
@@ -118,6 +118,28 @@ function Get-AllPackageInfoFromRepo ([string] $ServiceDirectory) {
   return $allPackageProps
 }
 
+function Get-rust-AdditionalValidationPackagesFromPackageSet ($packagesWithChanges, $diff, $allPackageProperties) {
+  # if the change was in a service directory, but not in a package directory, test all the packages in the service directory
+  [array]$serviceFiles = ($diff.ChangedFiles + $diff.DeletedFiles) | ForEach-Object { $_ -replace '\\', '/' } | Where-Object { $_ -match "^sdk/.+/" }
+  # remove files that target any specific package
+  foreach ($package in $allPackageProperties) {
+    $serviceFiles = $serviceFiles | Where-Object { "$RepoRoot/$_" -notmatch "^$($package.DirectoryPath)/" }
```

Comment:
If you had more than 1 package, wouldn't this remove everything not found in the first?

Reply:
x = x not like y
x = x not like z

is the same as

x = x not like y and not like z